docs/System-Requirements.md

```diff
@@ -60,3 +60,22 @@ You need following software versions:
 Deployment is described in a separate documentation:
 
 - [Docker Images for PowerAuth](https://github.com/wultra/powerauth-docker)
+
+
+## Entropy
+
+The PowerAuth stack is heavy on entropy because of random number generators (RNG) used for cryptography.
```

Comment:
Can we change `heavy on entropy` to `requires significant amount of entropy`?

Reply:
Sure, thanks for the comment.